python-package/securexgboost/core.py

```diff
@@ -355,6 +355,14 @@ def _maybe_dt_array(array):
 
     return array
 
+def switch_user(user_name):
```

Comment:
Let's change the name of this function to `set_user`, though I see why you called it `switch_user`. In the outsourced computation model, each client has one username, and shouldn't need to switch user midway through computation. 

I think changing the name to `set_user` should alleviate any potential confusion in the outsourced computation model while maintaining clarity in your example. Thoughts?

If you change the name please remember to also change the corresponding code in `python-package/securexgboost/__init__.py`

Reply:
make sense I agree. Just did a replace all.